### PR TITLE
Fix `activeElement` for ShadowRoot: focused elements in light DOM of host are now returned.

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

**Comment by bicknellr** [comment]
This doesn't handle the case that the active element is within the shadow DOM of an element in the ShadowRoot's host's light DOM.

**Comment by bicknellr** [comment]
Ok, fixed for that case.

**Comment by bicknellr** [comment]
@azakus ping

File: src/ShadowDOM/wrappers/ShadowRoot.js [hunk 1]
```diff
@@ -80,8 +80,13 @@
 
       var activeElement = wrap(unwrappedActiveElement);
 
-      // Loop while activeElement is not a shallow child of this ShadowRoot.
-      while (!this.contains(activeElement)) {
+      if (activeElement === this.host) {
```
Comment (azakus) [inline src/ShadowDOM/wrappers/ShadowRoot.js]: Please add a comment about how the host will not be considered the "activeElement" of a shadowroot.

**Comment by azakus** [comment]
LGTM with nit

File: src/ShadowDOM/wrappers/ShadowRoot.js (hunk 1 above)
Comment (bicknellr) [inline src/ShadowDOM/wrappers/ShadowRoot.js]: Added.

**Comment by bicknellr** [comment]
@azakus I think the failing tests are unrelated again - are these still expected for the time being?

**Comment by azakus** [comment]
LGTM!

**Pull request merged**